deformable_manipulation_experiment_params/launch/cloth_single_pole_params.launch

```diff
@@ -127,5 +132,10 @@
         <group ns="jacobian_controller">
             <param name="collision_scaling_factor"      value="1000.0"          type="double"/>
         </group>
+
+        <group ns="sampling_controller">
```

Comment:
Is this still only for a sampling controller, or is the nomad controller used here?